mutagen/ogg.py

```diff
@@ -81,6 +81,12 @@ def __init__(self, fileobj=None):
         self.offset = fileobj.tell()
 
         header = fileobj.read(27)
+
+        # If there is not enough data to make up the header...
+        # we might be looking at trailing null bytes on the file.
+        if len(header) != 27 and all(byte == 0 for byte in header):
```

Comment:
This works for most cases, but I see two potential issues here:

1. A file with exactly 27 trailing null bytes at the end will fail as before.
2. A file with 27 or more null bytes will also fail, but we don't know what comes after the 27 bytes. If it is only null bytes we could probably treat it as EOF as well. If there comes other garbage data failing seems correct.

Just wanted to mention this. Not sure how to handle this all. Probably your check is fine. This will solve the more common case of having only a few null bytes at the end. Everything above 26 null bytes we play save and fail as before.

I would just suggest to change the check to `len(header) < 27`. The result is the same, as only max. 27 bytes got read. But to me it makes the intent clearer.